hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/ratis/RatisHelper.java

```diff
@@ -277,9 +278,15 @@ static RetryPolicy createRetryPolicy(Configuration conf) {
         .toIntExact(TimeUnit.MILLISECONDS), TimeUnit.MILLISECONDS);
     TimeDuration sleepDuration =
         TimeDuration.valueOf(retryInterval, TimeUnit.MILLISECONDS);
-    RetryPolicy retryPolicy = RetryPolicies
+    RetryPolicy writeRetryPolicy = RetryPolicies
         .retryUpToMaximumCountWithFixedSleep(maxRetryCount, sleepDuration);
-    return retryPolicy;
+    final RequestTypeDependentRetryPolicy.Builder builder =
+            RequestTypeDependentRetryPolicy.newBuilder();
+    builder.set(RaftProtos.
+            RaftClientRequestProto.TypeCase.WRITE, writeRetryPolicy);
+    builder.set(RaftProtos.
+            RaftClientRequestProto.TypeCase.WATCH, RetryPolicies.noRetry());
```

Comment:
I think we will need to change the watch timeout as well in the leader(default is 10sec, we dont have a config for it in ozone). Maybe we should keep 2-3 retries to include the case of leader changes.